noble/noble.d.ts

```diff
@@ -1,6 +1,7 @@
 // Type definitions for noble
 // Project: https://github.com/sandeepmistry/noble
 // Definitions by: Seon-Wook Park <https://github.com/swook>
+// Additions by: Hans Bakker <https://github.com/wind-rider>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
```

Comment:
also - when I install the typings for node they are in /// \<reference path="../node/**index**.d.ts" />. Could you have a look at that?